src/pull.go

```diff
@@ -51,8 +52,47 @@ func (g *Commands) Pull(byId bool) error {
 	cl, clashes, err := pullLikeResolve(g, byId)
 
 	if len(clashes) >= 1 {
-		warnClashesPersist(g.log, clashes)
-		return ErrClashesDetected
+		if g.opts.FixClashes {
+			g.log.Logln("Some clashes found")
+			if g.opts.canPrompt() {
+				if !promptForChanges("Try fixing them [Y/N] ? ") {
+					return ErrClashesDetected
+				}
+			}
+			sort.Sort(ByPath(clashes))
+
+			lastPath := ""
```

Comment:
Great suggestion. For sure, please do. Once this PR is merged in, I'll move the Rename code in there too, to avoid creating noise for this PR.